src/test/tests.ts

```diff
@@ -133,22 +127,22 @@ const subscriptionManager = new SubscriptionManager({
 
 // indirect call to support spying
 const handlers = {
-  onSubscribe: (msg: SubscribeMessage, params: SubscriptionOptions, webSocketRequest: WebSocket) => {
-    return Promise.resolve(Object.assign({}, params, {context: msg['context']}));
+  onSubscribe: (msg: OperationMessage, params: SubscriptionOptions, webSocketRequest: WebSocket) => {
```

Comment:
yeah but we need to "translate" it for legacy clients, just like we do with the callback itself of the data.